samples/sampleError/template.json

```diff
@@ -0,0 +1,274 @@
+{
```

Comment:
remove `sampleError` folder. Will cover in separte pytest PR

Reply:
Deleted